[MC-653] New workflow - Quarantine Host in FireEye HX from Slack (#527)
rapid7/insightconnect-workflows: workflows/Quarantine_Host_in_FireEye_HX_from_Slack/help.md

# Description

This workflow quarantines the host in FireEye HX using the Slack command and sends the information back to Slack.

Sample Slack Trigger Command:

`@Rapid7 InsightConnect quarantine-host example_host`

# Key Features

* Quarantine hosts

# Requirements

* [Slack](https://insightconnect.help.rapid7.com/docs/configure-slack-for-chatops)
* FireEye HX credentials

# Documentation

## Setup

Import the workflow from the Rapid7 Extension Library and proceed through the Import Workflow wizard in InsightConnect. Import plugins, create or select connections, and rename the workflow as a part of the Import Workflow wizard as necessary.

This workflow leverages InsightConnect's Parameters feature. This feature allows variables used multiple times throughout a workflow to be entered once and then referenced throughout the workflow.

> There is one parameter you will need to configure in order to complete setup of your workflow:

Reviewer: ```suggestion
There are two parameters you will need to be configured in order to complete setup of your workflow:
```

> * Slack Channel: The Slack channel name in your environment where the workflow should be triggered and respond

Reviewer: ```suggestion
* Slack Channel: The Slack channel name in your environment where the workflow should be triggered from and where it responds to
```
